internal/librarian/librarian.go

```diff
@@ -65,11 +77,14 @@ func newLibrarianCommand() *cli.Command {
 }
 
 func newCmdGenerate() *cli.Command {
+	var verbose bool
```

Comment:
Is there someplace more upstream we can make this change so we don't accidently forget to write this code as we add more commands in the future.

Reply:
Yes, I had a first version done before newLibrarianCommand() here. But it happens before flag parsing and needs manual check on args.
https://github.com/googleapis/librarian/blob/4554b399378313d5dc802c26795ed66e52c9ef6d/internal/librarian/librarian.go#L27-L30

I find it a bit error prone and brittle that it works around the cli framework, and it still needs to add flag to each command for documentation purpose.

Reply:
ack